docs/sidebars.js

```diff
@@ -4,102 +4,133 @@ const sidebars = {
       type: "category",
       label: "ZIO HTTP",
       collapsed: true,
+
+      // main documentation index
       link: { type: "doc", id: "index" },
       items: [
-        "installation",
+        "installation", 
+
+
+        // Reference section
         {
           type: "category",
           collapsed: true,
           link: { type: "doc", id: "reference/index" },
           label: "Reference",
           items: [
-            "reference/server",
-            "reference/client",
+            "reference/overview",
+            "reference/server", 
+            "reference/client", 
+            "reference/handler",
```

Comment:
I would suggest placing the handler after the routing section for better organization.